package.json

```diff
@@ -35,6 +35,7 @@
     ],
     "devDependencies": {
         "@fec/remark-a11y-emoji": "^3.1.0",
+        "@iconify/react": "^3.0.1",
```

Comment:
No package should be needed for this fix

let me know if you need more context.